metricbeat/mb/registry.go

```diff
@@ -0,0 +1,141 @@
+package mb
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+const initialSize = 10 // initialSize specifies the initial size of the Register.
```

Comment:
Should we set this to 1? As the list is only created once it should not be a big overhead to extend it. I assume there will be quite a few uses cases where only 1 module and metricset are used.